README.md

```diff
@@ -85,32 +93,49 @@ packages for the agent.
 For non-containerized environments, there is a convenience script that you can
 run on your host to install the agent package.  This is useful for testing and
 trials, but for full-scale deployments you will probably want to use a
-configuration management system like Chef or Puppet.  You can [view the source
-for the installer
-script](./deployments/installer/install.sh)
+configuration management system like Chef or Puppet.
+
+##### Linux
+You can [view the source for the installer script](./deployments/installer/install.sh)
 and use it on your hosts by running:
 
 ```sh
 curl -sSL https://dl.signalfx.com/signalfx-agent.sh > /tmp/signalfx-agent.sh
 sudo sh /tmp/signalfx-agent.sh YOUR_SIGNALFX_API_TOKEN
 ```
 
+##### Windows
+The Agent has two dependencies on Windows which must be satisfied before running the installer script.
+
+- [.Net Framework 3.5](https://docs.microsoft.com/en-us/dotnet/framework/install/dotnet-35-windows-10) (Windows 8+)
+- [Visual C++ Compiler for Python 2.7](https://www.microsoft.com/EN-US/DOWNLOAD/DETAILS.ASPX?ID=44266)
+
+The installer script is written for Powershell v3.0 and above and will not function correctly on earlier versions.
```

Comment:
Is there a reason this isn't just listed as a third dependency, for consistency in formatting?